File: java/client/src/main/java/org/signal/libsignal/metadata/certificate/InvalidCertificateException.java
//
// Copyright 2023 Signal Messenger, LLC.
// SPDX-License-Identifier: AGPL-3.0-only
//

package org.signal.libsignal.metadata.certificate;

public class InvalidCertificateException extends Exception {
  public InvalidCertificateException(String s) {
    super(s);
  }

  public InvalidCertificateException(Exception e) {
    super(e);
  }
}

File: java/shared/java/org/signal/libsignal/metadata/certificate/InvalidCertificateException.kt
//
// Copyright 2023 Signal Messenger, LLC.
// SPDX-License-Identifier: AGPL-3.0-only
//

package org.signal.libsignal.metadata.certificate

public class InvalidCertificateException : Exception {
  public constructor(s: String) : super(s)

  public constructor(e: Exception) : super(e)
}
